README.md

```diff
@@ -21,8 +25,15 @@ var results = AuthorModel.find({name: "Bob"});
 console.log(results);
 ```
 
+```text
+find returns a list of documents, not a document. Even if there's only one bob, it will return a one item list 
+containing bob the document, (If this was my family, it would return my brother-in-law Bob, my Uncle Bob on 
+Mom's side, his son Bob, my Uncle Bob on Dad's side, and maybe his son Rob and the Bob I dated a little.)
+```
+
 ```js
-// Your answer...
+var results = AuthorModel.findOne({name: "Bob"});
+console.log(results);
 ```
```

Comment:
The issue is that `findOne` is asynchronous, meaning that if we are going to `console.log(results)` we must do that inside of a callback or promise.